topicctl_parser/notifications/datadog.py

```diff
@@ -0,0 +1,37 @@
+import click
+import time
+import httpx
+import os
+
+from typing import Any, Dict
+
+
+DD_API_BASE = "https://api.datadoghq.com"
+
+DATADOG_API_KEY = os.getenv("DATADOG_API_KEY") or os.getenv("DD_API_KEY")
+assert DATADOG_API_KEY, "Please set either DATADOG_API_KEY or DD_API_KEY env var."
+
+
+def send_event_payload_to_datadog(payload: Dict[str, Any]) -> None:
```

Comment:
`payload: Mapping[str, Any]` 

You do not need to be specific on the attribute type, Anything that implements the Mapping interface would work. Using the abstract interface rather than the specific implementation makes your function easier to reuse as it does not ask for something it does not need.